iOS/Views/NonSelectableView.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Created for schulcloud-mobile-ios under GPL-3.0 license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import UIKit
+
+class NonSelectableView: UIView {
```

Comment:
I find this name a bit misleading because there is no code regarding selection.
How about `FixedBackgroundColorView` or somethings similar?